checkin/models.py

```diff
@@ -11,12 +18,28 @@ def __str__(self):
         return self.title
 
     def locale_title(self, language_code):
+        """
+        Finds the title of the skill in the given language
+
```

Comment:
Combine these two sentences, both describing the method, into one line. If the line becomes longer than 120 characters (counting each tab as 4 characters), place the remaining content on the next line.